metron-platform/metron-enrichment/src/test/java/org/apache/metron/enrichment/integration/EnrichmentIntegrationTest.java

```diff
@@ -160,16 +159,36 @@ public void test() throws Exception {
       fluxComponent.submitTopology();
 
       kafkaComponent.writeMessages(Constants.ENRICHMENT_TOPIC, inputMessages);
-      List<Map<String, Object>> docs = runner.process(getProcessor(inputMessages));
-      Assert.assertEquals(inputMessages.size(), docs.size());
-      List<Map<String, Object>> cleanedDocs = docs;
-      validateAll(cleanedDocs);
-    }
-    finally {
+      ProcessorResult<List<Map<String, Object>>> result = runner.process(getProcessor(inputMessages));
+      List<Map<String, Object>> docs = result.getResult();
+      if (result.failed()){
+        result.printBadResults();
+        System.out.println(String.format("%d Valid Messages Processed", docs.size()));
```

Comment:
So, print to err and not out in the printBadResults()?  I discuss the accumulating function below